guides/user-guides/message-actions.md

```diff
@@ -30,17 +30,17 @@ Starred messages are per chat room and are not global; each room has its list of
 
 ## Pin messages
 
-Pin a message to a channel to make that message easily accessible to all user's on the channel. To pin a message, select the **Pin Message** option in the message actions menu. Remove already pinned messages by selecting **Remove Pin**.
+Pin a message to a channel to make that message easily accessible to all users on the channel. To pin a message, select the **Pin Message** option in the message actions menu. Remove already pinned messages by selecting **Remove Pin**.
 
-_The administrator of a Rocket.Chat team sets the permissions for who can pin messages in a channel. By default, only administrators, moderators, and owners of the channel can pin messages ._
+_The administrator of a Rocket.Chat team sets the permissions for who can pin messages in a channel. By default, only administrators, moderators, and owners of the channel can pin messages._
```

Comment:
If we are being picky then the comma before the 'and' is unnecessary in my opinion. I won't block the approval of this PR on this though.

Reply:
That's the Oxford Comma Style. Personally, I'm a fan of that style, but in any case... I didn't touch the comma, so it's unrelated to this PR.